examples/js/archive.js

```diff
@@ -174,8 +187,12 @@ tileMap.addEventListener('click', (event) => {
 
 document.addEventListener('click', (event) => {
   if (event.target.classList.contains('place-button')) {
-    const imageURL = event.target.previousElementSibling.dataset.original;
-    const image = L.distortableImageOverlay(imageURL);
+    const imageURL = event.target.previousElementSibling.src;
+    image = L.distortableImageOverlay(
+        imageURL,
+        {tooltipText: imageTooltipText}
```

Comment:
If we used the filename, we could just split `event.target.previousElementSibling.src` with `split('/')` and take any text after the final `/` - saving us a lot of complexity above, see?

Reply:
Nice.... I'll give this a shot. Many thank!